### PR TITLE
feat: create job card component relates #41

### PR DESCRIPTION
**Pull request by shathakh**

### CODE REVIEW

File: client/src/components/jobCard/jobCard.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            </div>
+            <div className="budget">
+              budget:
+              <span>$50</span>
+            </div>
+          </div>
+          <Link to="/job/jobid">
+            <Button
+              className="apply-job-btn"
+              style={{
```
Comment (LinaYahya) [inline client/src/components/jobCard/jobCard.tsx:35]: It'll be exhausting to us to style every button in the app, instead, we can create an app theme in the `app` and change colors `primary` and `secondray` and pass theme to the app, Then all of our app will apply that theme
Check [createTheme API](https://mui.com/material-ui/customization/theming/#createtheme-options-args-theme)

File: client/src/components/jobCard/jobCard.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <div className="proposals">
+              proposals:
+              <span>3</span>
+            </div>
+            <div className="budget">
+              budget:
+              <span>$50</span>
+            </div>
+          </div>
+          <Link to="/job/jobid">
```
Comment (LinaYahya) [inline client/src/components/jobCard/jobCard.tsx:32]: why not to add `onClick` button handler and then navigate to the desired route
check [useNavigate API](https://reactrouter.com/en/main/hooks/use-navigate)

File: client/src/components/jobCard/jobCard.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import './jobCard.css';
+
+function JobCard() {
+  return (
+    <div className="content">
+      <div className="job-card">
+        <Link to="/job/jobid">
+          <h2 className="h2">
```
Comment (LinaYahya) [inline client/src/components/jobCard/jobCard.tsx:12]: I would prefer to use typography mui component, Even If we want to keep use `h2` try to change `className`

File: client/src/components/jobCard/jobCard.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import {
+  Button,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import './jobCard.css';
```
Comment (LinaYahya) [inline client/src/components/jobCard/jobCard.tsx:5]: rename files to `index.ts` and `style.css`

**Review by LinaYahya — CHANGES_REQUESTED** [review]
Great Job Shatha!
Just small notes to take

**Review by mohammedagha27 — APPROVED** [review]

**Review by alishanti98 — APPROVED** [review]

**Review by naghamabuwarda — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**